arches_lingo/src/arches_lingo/components/tree/ConceptTree.vue

```diff
@@ -401,4 +406,14 @@ function onNodeSelect(node: TreeNode) {
 :deep(.p-tree-filter-input) {
     border-radius: 0.125rem;
 }
+
+.skeleton {
+    padding: var(--p-tree-padding);
+    width: 29rem;
```

Comment:
with luck you shouldn't need to hardcode the skeleton width, maybe drawer width?